CMakeLists.txt

```diff
@@ -343,6 +344,36 @@ if(LOVR_ENABLE_AUDIO)
     src/api/l_audio_source.c
     src/lib/miniaudio/miniaudio.c
   )
+
+  if(LOVR_USE_OCULUS_AUDIO)
+    if(NOT LOVR_OCULUS_AUDIO_PATH)
+      message(FATAL_ERROR "LOVR_USE_OCULUS_AUDIO requires the LOVR_OCULUS_AUDIO_PATH to be set to the location of the Oculus Spatializer Native (AudioSDK) folder")
+    endif()
+
+    include_directories("${LOVR_OCULUS_AUDIO_PATH}/Include")
+
+    add_library(OculusAudio SHARED IMPORTED)
+    if (ANDROID)
+      set_target_properties(OculusAudio PROPERTIES IMPORTED_LOCATION "${LOVR_OCULUS_AUDIO_PATH}/Lib/Android/arm64-v8a/libovraudio64.so")
+    elseif(WIN32) # Note: This has *not* been tested.
+      if (CMAKE_GENERATOR_PLATFORM STREQUAL "x64")
+        set_target_properties(OculusAudio PROPERTIES IMPORTED_IMPLIB "${LOVR_OCULUS_AUDIO_PATH}/Lib/x64/ovraudio64.lib")
+        set_target_properties(OculusAudio PROPERTIES IMPORTED_LOCATION "${LOVR_OCULUS_AUDIO_PATH}/Lib/x64/ovraudio64.dll")
+      else () # ARM and ARM64 are also valid values for CMAKE_GENERATOR_PLATFORM but we will ignore this
+        set_target_properties(OculusAudio PROPERTIES IMPORTED_IMPLIB "${LOVR_OCULUS_AUDIO_PATH}/Lib/Win32/ovraudio32.lib")
+        set_target_properties(OculusAudio PROPERTIES IMPORTED_LOCATION "${LOVR_OCULUS_AUDIO_PATH}/Lib/Win32/ovraudio32.dll")
+      endif()
+    elseif (APPLE) # This *might* be possible-- there is a .framework in "macub/" ("universal binary"?)
+      message(FATAL_ERROR "LOVR_USE_OCULUS_AUDIO is not currently supported on Apple platforms")
+    else() # Assume Linux. Note: This has *not* been tested. FIXME: When is the .so copied?
+      set_target_properties(OculusAudio PROPERTIES IMPORTED_LOCATION "${LOVR_OCULUS_AUDIO_PATH}/Lib/Linux64/libovraudio64.so")
+    endif()
+
+    set(LOVR_OCULUS_AUDIO OculusAudio)
+    add_definitions(-DLOVR_ENABLE_OCULUS_AUDIO)
+    target_sources(lovr PRIVATE src/modules/audio/spatializers/oculus_spatializer.c)
+    target_link_libraries(lovr ${LOVR_OCULUS_AUDIO}) # TODO: Reorder?
+  endif()
```

Comment:
This should follow the same pattern for 3rd party libraries where it goes up above the lovr target and sets the `LOVR_OCULUS_AUDIO` var to the target (I don't know *why* things are like this but...should at least be consistent about it).  The variable can be added to the big list of targets on line 320ish.  The `target_sources` and `add_definitions` should stay here though.

For the FIXME about moving the .so on linux, don't worry about it.  Libraries aren't moved on linux and right now the rpath takes care of it.  I am going to do some relative rpath stuff soon so it matches macos though.

How does the library get moved into lib/arm64-v8a on android?

Reply:
"How does the library get moved into lib/arm64-v8a on android?" Uh... this is a good question. I think I haven't tested this specific patch on Android (the CMake code is copied from the previous version which WAS tested on android, but that was only tested on the old lovr-oculus-mobile system).